supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigWdlNamespace.scala

```diff
@@ -4,6 +4,7 @@ import com.typesafe.config.Config
 import cromwell.backend.impl.sfs.config.ConfigConstants._
 import net.ceedubs.ficus.Ficus._
 import wdl._
+import wom.core.WorkflowSource
```

Comment:
OOI, what here is still using `wdl._`?

Reply:
`WdlNamespace.loadUsingSource` in 2 spots